### DISCUSSION TITLE
Is Video CD being supported by dvdisaster ?

### DISCUSSION THREAD

**Discussion by Randname666** [speed47/dvdisaster#143] [Q&A] [answered]
I tried to recover a Video CD with dvdisaster and scanning marked all sectors as "illegal request: illegal mode for this track". I thought it's the disc is beyond repair in the first place, but after I tried another completely fine Video CD, it still reports all of its sectors as illegal requests.

So is Video CD being supported by dvdiaster ? If not, what makes Video CDs different than data CDs ?

**Comment by speed47** [comment, marked as answer]
A Video CD has an entirely different format (following the so-called "white book" standard) than a data CD (following the so-called "red book" standard), and is not supported by dvdisaster. 

Even if the medium appears to be identical ("it's just a CD!"), the way the data is engraved on it is completely different, and the dvdisaster logic can't apply to it, unfortunately.
